modules/upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -0,0 +1,88 @@
+[server-major-upgrade]
+= Server - Major Upgrade
+
+From time to time, {productname} Server upgrades are labeled as major.
+This is especially the case, if a version upgrade of openSUSE Leap is needed. openSUSE Leap is the underlying base operating system (OS).
+For background information about upgrading openSUSE Leap, see https://doc.opensuse.org/documentation/leap/startup/single-html/book-opensuse-startup/index.html#sec-update-zypper.
+
+Run the [command]``/usr/lib/susemanager/bin/server-migrator.sh`` script to upgrade the base OS and {productname} Server.
```

Comment:
Is this what you did during the testing? I am pretty sure you had to update a RPM package (susemanager) first.

Reply:
a package update was not available.  That's why I copy-pasted the commands on the command line (and replaced stable with master).

Reply:
Well, it wasn't because we didn't release yet, but *you need it* on a real situation :-)